### ISSUE TITLE
`visitable` doesn't encode dynamic segments

### ISSUE THREAD

**Issue by san650** [san650/ember-cli-page-object#202]
e.g.

``` js
const page = create({
  visit: visitable('/users/:id')
});

page.visit({ user: 'a/user' }); // => visits('/users/a/user');
```

The expected behavior is to navigate to `/users/a%2Fuser'` instead of `/users/a/user`

**Comment by san650** [comment]
I think we have to `encodeURIComponent` every dynamic segment value

**Comment by san650** [comment]
Fixed by #342
